### PR TITLE
Create six.scss

### PR DESCRIPTION
**Pull request by danielbarankin**
#### Hi! What is your PR about?

**- [ ] Add or edit a skin**
- [ ] Add a custom domain for your space
- [ ] Add an alias to migrate your space

### CODE REVIEW

File: skins/6ix.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    #topnav .router-link-active,
+    #topnav .logo {
+        font-size: 0;
+        line-height: 0;
+        width: 0;
+        height: 0;
+        display: none;
+    }
+}
```
Comment (gokaiorg) [inline skins/6ix.scss:18]: Please use only theme variables. No extra custom styles are allowed except for #topnav.
```suggestion
    #topnav {
        font-size: 0;
        line-height: 0;
        width: 0;
        height: 0;
        display: none;
    }
}
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
The colors used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme.
Capture d’écran 2021-11-13 à 16 07 30

**Comment by danielbarankin** [comment]
Please review, thank you :)

File: skins/six.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    --border-color: #eccece;
+    --header-bg: #131313;
+
+    #topnav {
+        font-size: 0;
+        line-height: 0;
+        width: 0;
+        height: 0;
+        display: none;
+    }
```
Comment (gokaiorg) [inline skins/six.scss:16]: Please use only colour styles, don't hide the top nav.
```suggestion
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Comment by danielbarankin** [comment]
Hi @gokaiorg , this PR is correctly configured. Fabien approved it. Please re-open and approve.

**Comment by bonustrack** [comment]
@danielbarankin Please make a single PR with all the required changes, this PR missing adding style in the index.js file. Also the changes on #topnav wont work well, did you tested it? This would hide the topnav, so there wont be way to connect wallet.

**Comment by danielbarankin** [comment]
Thank you cc’ing Adnan

**Pull request closed without merging**